src/python/grpcio/grpc/_server.py

```diff
@@ -719,6 +708,24 @@ def cancel_all_calls_after_grace():
   return shutdown_event
 
 
+def _start(state):
+  with state.lock:
+    if state.stage is not _ServerStage.STOPPED:
+      raise ValueError('Cannot start already-started server!')
+    state.server.start()
+    state.stage = _ServerStage.STARTED
+    _request_call(state)    
+    def cleanup_server(timeout):
+      if not timeout:
```

Comment:
Always favor
```
if <condition>:
  <other statements>
else:
  <some statements>
```
over
```
if not <condition>:
  <some statements>
else:
  <other statements>
```
. Except here I don't think it's a boolean comparison that best fits the situation: `timeout` may be a numeric value or it may be `None`, and I think the branch works best when aligned that way.